src/opensles/AudioStreamBuffered.cpp

```diff
@@ -40,19 +42,20 @@ void AudioStreamBuffered::allocateFifo() {
             capacity = getFramesPerBurst() * kDefaultBurstsPerBuffer;
             mBufferCapacityInFrames = capacity;
         }
-        mFifoBuffer.reset(new FifoBuffer(getBytesPerFrame(), capacity));
+        // TODO consider using std::make_unique if we require c++14
+        mFifoBuffer.reset( new FifoBuffer(getBytesPerFrame(), capacity));
     }
 }
 
 int64_t AudioStreamBuffered::getFramesWritten() {
-    if (usingFIFO()) {
+    if (mFifoBuffer) {
```

Comment:
Would prefer explicit check `mFifoBuffer != nullptr` to make it clear that `mFifoBuffer` isn't a bool.

Reply:
Ah right, I see later on you changed `mFifoBuffer` to a `unique_ptr` which has a bool operator. As you were :)

Reply:
I'm trying to be more C++ish. ;-)